01week/javascripting/data-types.js

```diff
@@ -0,0 +1,61 @@
+
+function returnDate(){
+  const date=new Date();
+  return date;}
```

Comment:
Const is redundant, can be simplified to one line. . `return new Data();` Also, ending bracket should be on it's own line